contents/docs/data-warehouse/query.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Query your sources
+sidebar: Docs
+showTitle: true
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+---
+
+import { ProductScreenshot } from 'components/ProductScreenshot'
+import DataWarehouseLight from '../../images/features/data-warehouse/dw-light.png'
+import DataWarehouseDark from '../../images/features/data-warehouse/dw-dark.png'
+import QueryLight from '../../images/features/data-warehouse/query-light.png'
+import QueryDark from '../../images/features/data-warehouse/query-dark.png'
+
+To see all of the sources available to query, go to the [data warehouse tab](https://us.posthog.com/data-warehouse). This page shows all the external and PostHog tables you can query through [SQL insights](/docs/product-analytics/sql) along with saved views. Click on any of the tables, and then click "Query" to open an SQL insight.
+
+<ProductScreenshot
+  imageLight={DataWarehouseLight} 
+  imageDark={DataWarehouseDark} 
+  alt="Data warehouse tab in PostHog app" 
+  classes="rounded"
+/>
+
+## Querying sources
+
+To start with querying your sources:
+
+1. Create a [new SQL insight](https://us.posthog.com/insights/new).
+2. Choose a table to query like `stripe_charge` or `hubspot_contacts`
+3. Write your SQL query using your table like `SELECT * FROM hubspot_contacts`
+4. Click "Update and run" to see the results.
+5. Modify your query using [SQL commands](/docs/product-analytics/sql) as needed to get the data you want like `SELECT email FROM hubspot_contacts WHERE city = 'Brisbane'`.
+6. Save your query.
+
+<ProductScreenshot
+    imageLight={QueryLight} 
+    imageDark={QueryDark} 
+    alt="Querying a source in PostHog" 
+    classes="rounded"
+  />
```

Comment:
broken image

Reply:
Huh, works for me 🤔 
![CleanShot 2024-02-27 at 08 27 48](https://github.com/PostHog/posthog.com/assets/34755028/abb5b053-ab41-446b-b04a-5d19ab40b54c)